package.json

```diff
@@ -24,6 +24,7 @@
     "stylelint:fix": "stylelint 'css/*.css' 'css/*.scss' 'src/**/*.scss' 'src/**/*.vue' --fix"
   },
   "dependencies": {
+    "16": "^0.0.2",
```

Comment:
Why is this dependency required?

Reply:
Actually, I was facing the npm version errors while running the **npm ci** or **npm build**.  The table view wasn't loading due to these errors. That's the reason for the above change.

Reply:
Does that mean the dependency isn't required?

Reply:
Yeah this should not be needed, can you drop the changes from package.json/package-lock.json from the PR?

Reply:
Yeah sure, let me do it.